client/src/main/java/io/pravega/client/stream/impl/ControllerResolverFactory.java

```diff
@@ -15,15 +15,21 @@
 import com.google.common.net.InetAddresses;
 import io.grpc.Attributes;
 import io.grpc.EquivalentAddressGroup;
+import io.grpc.LoadBalancerRegistry;
```

Comment:
Please change your formater to use our convention of putting all imports in alphabetical order followed by all static imports in alphabetical order.

Reply:
Done.